### DISCUSSION TITLE
Welcome to FragPipe-Analyst Discussions!

### DISCUSSION THREAD

**Discussion by hsiaoyi0504** [Nesvilab/FragPipe-Analyst-doc#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of FragPipe-Analyst community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aad100** [comment]
Hello,

My name is Adam, from the University of York UK.  Thank you very much for creating FragPipe-Analyst and shring it with the comunity.  It is much appriciated.  

I would just like to ask about data security.  Specifically if data is uploaded by a user to FragPipe-Analyst can anyone else see that data and are the uploded files stored or delteted when the user closes the browser.

Many thanks,

Adam

**Reply by hsiaoyi0504** [reply]
It's saved in the tmp directory on the server and deleted regularly. We don't manually check those files and would not provide them to anyone.
